src/App.tsx

```diff
@@ -9,6 +9,11 @@ import { PrivateRoute } from './components/PrivateRouter';
 import { ApolloProvider } from '@apollo/client';
 import { client } from './api';
 import { Logout } from './pages/Logout';
+import {Issues} from "./pages/Issues";
+import {PullRequests} from "./pages/PullRequests";
+import {Explore} from "./pages/Explore";
+import {Repositories} from "./pages/Repositories";
+import {Repository} from "./pages/Repository";
```

Comment:
Webstorm: auto import with single quotes and spases between braces
https://gist.github.com/zmts/662e25840ef29824d3e79db031b97745